projects/igniteui-angular/src/lib/snackbar/snackbar.component.html

```diff
@@ -2,6 +2,6 @@
     {{ textMessage }}
     <ng-content></ng-content>
 </div>
-<button class="igx-snackbar__button" igxRipple="white" *ngIf="actionText" (click)="triggerAction()">
+<button igxButton *ngIf="actionText" (click)="triggerAction()">
```

Comment:
@SisIvanova I see you're getting rid of the default ripple? Btw, it doesn't show because the import of the ripple is not included in the component, but if you add it, it would show. Does this button have a ripple by default according to spec? @simeonoff

Reply:
Since the configuration of the ripple is internal to the template, i.e. not configurable, we decided to get rid of it. However, we can make it configurable by providing in input, say `buttonRippleColor`, or by making the button templateable. In that way we can either assign the default value of the `buttonRippleColor` to be `transparent`, while still retaining the functionality or the users can just pass in their own button to which they can attach a ripple. What do you think?

Reply:
Let's get rid of it in the default template. Not to mention it's not working since `16.0.0` :). Being able to provide your own button covers the scenario of being able to also add `igxRipple` to it.

Reply:
It can be something of the sort:

```html
<ng-content #customButton select="[igxButton]"></ng-content>
<button *ngIf="!customButton" igxButton *ngIf="actionText" (click)="triggerAction()">{{actionText}}</button>
```